server/pbench/bin/index-pbench

```diff
@@ -763,9 +802,39 @@ class ToolData(object):
         return ts.strftime("%Y-%m-%dT%H:%M:%S.%f%z")
 
     def _make_source_unified(self):
-        """Process each .csv file at the same time, reading one row from each,
-        unifying them into one record per column identifier when the
-        timestamps match."""
+        """Create one JSON document per identifier, per timestamp from
+        the data found in multiple csv files.
+
+        This algorithm is only applicable to 2 or more csv files which
+        contain data about 1 or more identifiers.
+
+        The approach is to process each .csv file at the same time,
+        reading one row from each in lock step. The field data found
+        in each column across all files for a given identifier at the
+        same timestamp are unified into one JSON document.
+      
```

Comment:
White space.

Reply:
Fixing, and adding more documentation. =)